src/machine/machinectl.c

```diff
@@ -1119,11 +1127,12 @@ static int copy_files(int argc, char *argv[], void *userdata) {
                 host_path = abs_host_path;
         }
 
+
```

Comment:
Spurious newline here now?

Reply:
Ah perhaps I added it in wrong place from previous comment, let me check sorry about that one